src/utils/constants.js

```diff
@@ -29,9 +31,11 @@ export const STORAGE_TABS_RECENT_DAY_COUNT = 'tab.user.lastTabDay.count'
 export const STORAGE_TABS_LAST_TAB_OPENED_DATE = 'tab.user.lastTabDay.date'
 export const STORAGE_FEATURES = 'tab.user.features'
 export const MAX_TABS_WITH_ADS = 150
-
-// Constants for Local Storage Mgr
 export const HAS_SEEN_SEARCH_V2_TOOLTIP = 'has_seen_search_v2_tooltip'
+export const LAST_SEEN_GROUP_IMPACT_METRIC =
+  'tab.use.groupImpactMetric.lastSeenGroupImpactMetric'
+export const COMPLETED_GROUP_IMPACT_VIEWS =
```

Comment:
Did you want to add helpers for this too?

Reply:
Ah, I forgot to comment on these. I'm just predefining the constants I need, but I think the logic is simple enough that it's not worth writing helpers for. I'll add them later if I see fit.